AutocompleteClient/Constants/Constants.swift

```diff
@@ -77,6 +77,15 @@ public struct Constants {
         public static let defaultItemCountPerSection = 10
     }
 
+    public struct SearchQuery {
+        public static let pathString = "search"
+        public static let page = "page"
+        public static let groupFilter = "filters[group_id]"
+        public static let facetFilterKey = { (key: String) -> String in "filters[\(key)]" }
+        public static let numResultsPerPage = "num_results_per_page"
+        public static let numResultsPerPageForSectionNamePrefix = "num_results_per_page_"
```

Comment:
This parameter doesn't exist for search results... we can only have one section which defaults to Products, `section=Products`